repos/fdbt-site/src/components/RadioConditionalInput.tsx

```diff
@@ -357,6 +357,7 @@ const renderConditionalRadioButton = (
             type="radio"
             value={radio.value}
             data-aria-controls={radio.dataAriaControls}
+            defaultChecked={radio.defaultChecked}
```

Comment:
I dont think we should be changing this. This element is called 'uncheckedRadioInput' which means it should never be checked?